client/include/client.h

```diff
@@ -0,0 +1,57 @@
+#ifndef CLIENT_H
+#define CLIENT_H
+
+#include <netinet/in.h>
+#include <string>
+
+namespace cs    // client-server
```

Comment:
hmm. maybe two namespaces would be better?

Reply:
Done